public/Sjofn/practice/sjofn4.html

```diff
@@ -0,0 +1,32 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Practice 4</title>
+</head>
+<body>
+<!--<button onclick="instruction1()">Open console and click me</button>-->
```

Comment:
Please, remove this commented line. 
It is usually considered a bad practice to leave commented instructions/markup.